ports/libhv/portfile.cmake

```
@@ -17,6 +24,8 @@ vcpkg_cmake_configure(
         -DBUILD_UNITTEST=OFF
         -DBUILD_STATIC=${BUILD_STATIC}
         -DBUILD_SHARED=${BUILD_SHARED}
+        -DWITH_OPENSSL=ON
+        ${FEATURE_OPTIONS}
```

```suggestion
        ${FEATURE_OPTIONS}
```

Isn't this flag obsolete because of `${FEATURE_OPTIONS}`?